docs/ldml/tr35-keyboards.md

```diff
@@ -596,7 +585,7 @@ An element used to keep track of layout specific settings. This element may or m
 **Syntax**
 
 ```xml
-<settings [fallback="omit"] [transformFailure="omit"] [transformPartial="hide"] />
+<settings fallback="omit" normalization="mixed" />
```

Comment:
The brackets were denoting an optional attribute. You kept them above in the `info` element, it should be unified.